pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "appoinment-scheduler"
+version = "0.1.0"
+description = "Appoinment Scheduller API PET project"
+authors = ["Vasyl Boliuk <vbboliuk@gmail.com>"]
```

Comment:
@vasylboliuk since your repo is public, please remove your email from the source code. Public web scrappers are surfing internet for the emails to send you spam afterwards

Reply:
Done